redisgraph-bechmark-go.go

```diff
@@ -98,12 +102,15 @@ func main() {
 
 	dataPointProcessingWg.Add(1)
 	go processGraphDatapointsChannel(graphDatapointsChann, c1, *numberRequests, &dataPointProcessingWg, &instantHistogramsResetMutex)
-
+	clientTotalCmds := samplesPerClient
 	startTime := time.Now()
 	for client_id := 0; uint64(client_id) < *clients; client_id++ {
 		wg.Add(1)
 		rgs[client_id], conns[client_id] = getStandaloneConn(*graphKey, "tcp", connectionStr, *password)
-		go ingestionRoutine(&rgs[client_id], true, queries, cdf, samplesPerClient, *loop, *debug, &wg, useRateLimiter, rateLimiter, graphDatapointsChann)
+		if uint64(client_id) == (*clients - uint64(1)) {
```

Comment:
Please add a comment explaining this check, also explain why it is OK to update `clientTotalCmds` at this point

Reply:
:+1: addressed in latest commit.